packages/edit-site/src/components/post-edit/index.js

```diff
@@ -178,6 +178,7 @@ export function PostEdit( { postType, postId } ) {
 			} ) }
 			label={ __( 'Post Edit' ) }
 		>
+			<h2>{ __( 'Quick Edit' ) }</h2>
```

Comment:
It's best to use a Heading component instead of a hardcoded H2:
```
			<Heading level={ 2 } size={ 16 }>
				{ __( 'Quick Edit' ) }
			</Heading>
```